introduction.tex

```diff
@@ -1,3 +1,15 @@
+\section{Abstract}
+France is in pursuit of closing the fuel cycle by transitioning from a \gls{LWR} fleet
+to a \gls{SFR} one with recycling of \gls{SNF}. Past transition scenarios looked at
+deploying additional \glspl{LWR} in order to meet \gls{SNF} demands for \gls{MOX} production.
```

Comment:
This sentence, "Past ... production." doesn't seem to fit. I would recommend deleting it.